lib/rumbrl/ext/hash.rb

```diff
@@ -0,0 +1,8 @@
+# Extensions for native Hash to allow for log ingestion
+module HashExtensions
+  def to_log
+    to_hash
+  end
+end
+
+::Hash.include(HashExtensions)
```

Comment:
Anything. Any class that implements `to_log` allows for a special format. For example, say we have a user that has sensitive data that we want to pull out. Instead of writing something like

```
logger.info(message: "Some test", user: @user.as_json(except: [:first_name, :last_name, :other_sensitive_data]))
```

everytime we want to log it, we can do

```
class User
  def to_log
    as_json(except: [:first_name, :last_name, :other_sensitive_data])
  end
end

....

logger.info(message: "Some text", user: @user
```

and consolidate that knowledge to one spot without consumers (at the `logger.info` level) having to know about what to sanitize, or that `to_log` even exists.